yaml/restaurant_item.yml

```diff
@@ -0,0 +1,48 @@
+RestaurantItemEndpoint
+---
+get:
```

Comment:
add response schema, when I open swagger I should see: 1. what I need to send, request shape 2. what will be the response shape

Reply:
take a look at Flasgger examples, they should specify both request and response schema in their examples

Reply:
fixed